grafana/setup_grafana.go

```diff
@@ -78,8 +81,21 @@ func main() {
 		Password:  envParams["influxdb_password"],
 		Database:  envParams["influxdb_database"],
 	}
+	// Override setup env vars with Grafana configuration env vars if present
+	adminUser := envParams["grafana_user"]
+	if "" != envParams["gf_security_admin_user"] {
```

Comment:
May be better off using the `if val, ok := envParams[...]` construct (one less map access call).